microsoft-iis-mixin/signals/applications.libsonnet

```diff
@@ -0,0 +1,257 @@
+          prometheus: {
+            expr: 'windows_iis_total_worker_process_shutdown_failures{%(queriesSelector)s, app=~"$application"}',
+            rangeFunction: 'increase',
+            legendCustomTemplate: '{{instance}} - {{app}}',
+          },
+        },
+      },
+
+      workerProcessPingFailures: {
+        name: 'Worker process ping failures',
+        type: 'counter',
+        description: 'Worker process ping failures for an IIS application pool.',
+        unit: 'none',
+        sources: {
+          prometheus: {
+            expr: 'windows_iis_total_worker_process_ping_failures{%(queriesSelector)s, app=~"$application"}',
+            rangeFunction: 'increase',
+            legendCustomTemplate: '{{instance}} - {{app}}',
+          },
+        },
+      },
+
+      // Worker cache signals
+      workerFileCacheHitRatio: {
+        name: 'Worker file cache hit ratio',
+        type: 'raw',
+        description: 'The current file cache hit ratio for an IIS worker process.',
+        unit: 'percent',
+        sources: {
+          prometheus: {
+            expr: 'sum by(job, instance, app) (increase(windows_iis_worker_file_cache_hits_total{%(queriesSelector)s, app=~"$application"}[$__interval:]) / clamp_min(increase(windows_iis_worker_file_cache_queries_total{%(queriesSelector)s, app=~"$application"}[$__interval:]),1)) * 100',
+            legendCustomTemplate: '{{instance}} - {{app}}',
+          },
+        },
+      },
+
+      workerUriCacheHitRatio: {
+        name: 'Worker URI cache hit ratio',
+        type: 'raw',
+        description: 'The current URI cache hit ratio for an IIS worker process.',
+        unit: 'percent',
+        sources: {
+          prometheus: {
+            expr: 'sum by(instance, job, app) (increase(windows_iis_worker_uri_cache_hits_total{%(queriesSelector)s, app=~"$application"}[$__interval:]) / clamp_min(increase(windows_iis_worker_uri_cache_queries_total{%(queriesSelector)s, app=~"$application"}[$__interval:]),1)) * 100',
+            legendCustomTemplate: '{{instance}} - {{app}}',
+          },
+        },
+      },
+
+      workerMetadataCacheHitRatio: {
+        name: 'Worker metadata cache hit ratio',
+        type: 'raw',
+        description: 'The current metadata cache hit ratio for an IIS worker process.',
+        unit: 'percent',
+        sources: {
+          prometheus: {
+            expr: 'sum by(job, instance, app)(increase(windows_iis_worker_metadata_cache_hits_total{%(queriesSelector)s, app=~"$application"}[$__interval:]) / clamp_min(increase(windows_iis_worker_metadata_cache_queries_total{%(queriesSelector)s, app=~"$application"}[$__interval:]),1)) * 100',
+            legendCustomTemplate: '{{instance}} - {{app}}',
+          },
+        },
+      },
+
+      workerOutputCacheHitRatio: {
+        name: 'Worker output cache hit ratio',
+        type: 'raw',
+        description: 'The current output cache hit ratio for an IIS worker process.',
+        unit: 'none',
```

Comment:
Can we use the `percent` unit here? Since we're calculating a percentage anyway.